src/algebra/order/ring/lemmas.lean

```diff
@@ -548,7 +548,9 @@ lemma mul_lt_iff_lt_one_left
   a * b < b ↔ a < 1 :=
 iff.trans (by rw [one_mul]) (mul_lt_mul_right b0)
 
-/-! Lemmas of the form `1 ≤ b → a ≤ a * b`. -/
+/-! Lemmas of the form `1 ≤ b → a ≤ a * b`.
+
+Variants with `< 0` and `≤ 0` instead of `0 <` and `0 ≤` appear in `algebra/order/ring/defs`. -/
```

Comment:
I was about to ask whether you paid attention to keeping the sections structures parallel, but it seems you didn't. Why have them in different files?

Reply:
The lemma I need isn't available in this file. Much to my surprise, `defs` imports `lemmas` and not the other way around

Reply:
This is indeed very weird! What happened there?

Reply:
Oh I know, it's because `ring.lemmas` used to be called `monoid_with_zero_lemmas` while we still had a `ring` file. Still, I wonder whether we should turn the import around.

Reply:
Are you happy with things where they are for this PR then? How about I add something like
```suggestion
Variants with `< 0` and `≤ 0` instead of `0 <` and `0 ≤` appear in `algebra/order/ring/defs`
(which imports this file) as they need additional results proven there. -/
```